README.md

```diff
@@ -24,28 +25,43 @@ The file should have the following format:
         "password": false,
         "db": "database_to_backup"
       },
+      "mysql": {
+        "username": "root",
+        "password": "",
+        "host": false,
+        "port": false,
+        "db": "database_to_backup"
+      },
       "s3": {
         "mongo": {
           "key": "your_s3_key",
           "secret": "your_s3_secret",
           "bucket": "s3_bucket_to_upload_to",
           "destination": "/mongo",
-          "encrypt": true,
+          "encrypt": true
         },
         "redis": {
           "key": "your_s3_key",
           "secret": "your_s3_secret",
           "bucket": "s3_bucket_to_upload_to",
           "destination": "/redis",
-          "encrypt": true,
+          "encrypt": true
+        },
+    
+        "mysql": {
+          "key": "your_s3_key",
+          "secret": "your_s3_secret",
+          "bucket": "s3_bucket_to_upload_to",
+          "destination": "/mysql",
+          "encrypt": true
         }
       },
       "cron": {
-        "time": "11:59",
+        "crontab": "* * * * * *"
```

Comment:
Actually I did not change that. I guess I pulled the code from the other branch and sent a PR for the master branch